docs(ai-skills): add scope and cross-agent install guidance (#45583)
supabase/supabase: apps/docs/content/guides/getting-started/mcp.mdx

For curated, ready-to-use prompts that work well with IDEs and AI agents, see our [AI Prompts](/docs/guides/getting-started/ai-prompts) collection.

> To also install Supabase agent skills alongside the MCP server, use the [Supabase agent plugin](/docs/guides/getting-started/plugins) for a combined one-step setup.

Reviewer: ```suggestion
Additionally, you can install Supabase agent skills alongside the MCP server, use the [Supabase agent plugin](/docs/guides/getting-started/plugins) for a combined one-step setup.
```
